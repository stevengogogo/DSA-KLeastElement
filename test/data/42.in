10 50000
-8874 8525 -3518 8112 8319 5127 9907 9572 -8349 8620
Query 9 9 1
Insert 2 -3215
Insert 10 43
Insert 6 9651
Reverse 8 13
Reverse 2 6
Insert 10 7606
Reverse 9 9
Reverse 7 7
Reverse 1 2
Delete 8
Insert 12 -4923
Insert 5 9705
Query 12 13 1
Insert 12 -7687
Query 2 5 2
Delete 4
Delete 12
Insert 3 -5855
Query 12 14 3
Insert 13 3169
Query 15 16 2
Reverse 14 14
Insert 5 4338
Query 4 11 1
Delete 7
Query 8 13 2
Reverse 14 15
Delete 13
Delete 5
Delete 4
Query 6 13 8
Insert 12 -9275
Reverse 5 12
Delete 7
Reverse 6 7
Insert 2 3413
Insert 12 7068
Delete 13
Reverse 3 5
Query 10 13 2
Delete 5
Query 12 13 1
Delete 13
Delete 4
Reverse 6 10
Reverse 3 7
Query 7 8 1
Insert 9 2820
Delete 9
Reverse 1 4
Query 4 10 1
Reverse 6 7
Delete 1
Reverse 4 5
Query 5 7 2
Query 1 10 8
Query 4 9 6
Delete 1
Reverse 8 9
Reverse 8 8
Reverse 5 6
Query 1 4 3
Query 5 5 1
Query 9 9 1
Query 4 6 1
Reverse 8 8
Query 8 8 1
Delete 5
Query 7 8 2
Delete 7
Query 5 7 3
Reverse 3 6
Query 4 7 2
Insert 6 2853
Reverse 3 6
Delete 8
Reverse 5 5
Insert 8 -9949
Delete 5
Reverse 7 7
Reverse 6 7
Query 2 3 1
Insert 4 6154
Query 2 7 1
Insert 9 7161
Query 7 8 2
Reverse 6 6
Insert 10 -2882
Delete 9
Insert 2 -588
Insert 9 9485
Insert 6 -2440
Reverse 6 7
Reverse 12 12
Insert 5 2641
Insert 1 -1363
Insert 13 2107
Delete 7
Query 6 14 9
Reverse 3 4
Query 7 8 1
Insert 12 -8691
Reverse 5 11
Query 3 15 3
Insert 8 7647
Delete 3
Delete 9
Insert 8 -7407
Query 7 7 1
Delete 8
Reverse 12 12
Query 6 14 8
Reverse 5 12
Delete 6
Delete 2
Delete 10
Query 10 11 1
Delete 6
Query 3 5 2
Delete 3
Reverse 8 8
Insert 1 388
Query 8 8 1
Reverse 6 10
Query 5 9 5
Delete 8
Reverse 5 9
Reverse 2 3
Reverse 5 8
Delete 9
Reverse 5 6
Delete 7
Insert 4 -4223
Delete 1
Insert 8 3873
Query 1 6 1
Reverse 4 8
Query 8 8 1
Reverse 4 7
Query 4 8 5
Insert 2 1230
Delete 9
Reverse 5 5
Delete 2
Delete 5
Query 4 6 3
Insert 1 -6766
Delete 7
Insert 4 4641
Delete 7
Delete 6
Query 2 4 2
Query 1 2 2
Insert 1 2962
Reverse 1 5
Delete 6
Query 2 5 4
Reverse 1 2
Insert 6 -5232
Insert 6 -7700
Query 1 7 4
Reverse 1 4
Delete 1
Insert 7 1658
Delete 6
Query 6 6 1
Reverse 2 2
Reverse 3 6
Reverse 2 2
Reverse 2 4
Insert 6 6355
Reverse 2 7
Reverse 4 6
Delete 1
Query 4 6 3
Delete 2
Delete 4
Query 1 4 3
Reverse 4 4
Query 2 2 1
Query 2 3 1
Delete 2
Reverse 1 3
Reverse 3 3
Reverse 1 1
Delete 1
Delete 1
Delete 1
Insert 1 -6482
Delete 1
