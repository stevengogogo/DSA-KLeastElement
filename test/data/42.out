-8349
-4923
-3518
9907
9907
-8349
-7687
9907
-4923
-4923
-8349
-9275
-8349
9651
9907
9651
-8349
9907
-9275
-4923
9907
9907
43
2853
-9949
9705
9705
-2440
-2882
7647
7647
-2882
2853
7606
9705
-4223
2107
9705
9705
-588
-588
2962
-1363
1658
4641
1658
1658
1658
